package.json

```diff
@@ -61,12 +61,15 @@
     "pack-client": "npm-run-all -p -l pack-client-*",
     "test-client-template": "cd sdk/template/template && npm run test",
     "test-client": "npm-run-all -p -l \"test-client-* -- {@}\"",
-    "live-test-client-eventhubs-client": "cd packages/@azure/eventhubs/client && npm run unit",
-    "live-test-client-eventhubs-processor": "cd packages/@azure/eventhubs/processor && npm run unit",
-    "live-test-client-servicebus": "cd packages/@azure/servicebus/data-plane && npm run unit",
-    "live-test-client-storage-blob": "cd packages/@azure/storage/blob && npm run unit",
-    "live-test-client-storage-file": "cd packages/@azure/storage/file && npm run unit",
-    "live-test-client-storage-queue": "cd packages/@azure/storage/queue && npm run unit",
+    "live-test-client-eventhubs-client-node": "cd packages/@azure/eventhubs/client && npm run unit",
+    "live-test-client-eventhubs-processor-node": "cd packages/@azure/eventhubs/processor && npm run unit",
+    "live-test-client-servicebus-node": "cd packages/@azure/servicebus/data-plane && npm run unit",
+    "live-test-client-storage-blob-node": "cd packages/@azure/storage/blob && npm run unit-node",
+    "live-test-client-storage-file-node": "cd packages/@azure/storage/file && npm run unit-node",
+    "live-test-client-storage-queue-node": "cd packages/@azure/storage/queue && npm run unit-node",
+    "live-test-client-storage-blob-browser": "cd packages/@azure/storage/blob && npm run unit-browser",
+    "live-test-client-storage-file-browser": "cd packages/@azure/storage/file && npm run unit-browser",
+    "live-test-client-storage-queue-browser": "cd packages/@azure/storage/queue && npm run unit-browser",
```

Comment:
Please add no-op scripts for eventhubs and servicebus browser tests, like so:

```
"live-test-client-eventhubs-client-browser": "echo skipped",
"live-test-client-eventhubs-processor-browser": "echo skipped",
"live-test-client-servicebus-browser": "echo skipped",
```

Reply:
Done